app/src/main/java/fr/neamar/kiss/broadcast/IncomingSmsHandler.java

```diff
@@ -27,6 +27,8 @@ public void onReceive(Context context, Intent intent) {
             return;
         }
 
+        contactsProvider.reload();
```

Comment:
Hum... this means that we reload contacts on every message?

However, this is a costly process (up to 4 seconds), I'm not sure we want to run that every time :(